test/10-verify.spec.js

```diff
@@ -190,6 +190,27 @@ describe('vc.issue()', () => {
     error.message.should
       .contain('"suite.verificationMethod" property is required.');
   });
+  it('should add issuanceDate to verifiable credentials', async () => {
+    const credential = jsonld.clone(mockCredential);
+    delete credential.issuanceDate;
+    const now = new Date();
+    const jsonDate = now.toJSON();
+    const expectedIssuanceDate = `${jsonDate.slice(0, jsonDate.length - 5)}Z`;
```

Comment:
Just another way of doing it, but one less line.
```suggestion
        const expectedIssuanceDate = now.toISOString().slice(0, -5) + 'Z';
```

Reply:
I like the `now.toISOString()` approach better, but we should keep the string interpolation in place.

Reply:
@JSAssassin @BigBlueHat I like the `toISOString` method more too, I was copying to the way the issuanceDate is set inside of `index.js` should I update that to? test updated here: https://github.com/digitalbazaar/vc/pull/166/commits/b34181d36974aa88795319bb6d997d6dad6efc62

p.s. I could update the one in `index.js` to use `toISOString.slice(0, -5)` in the `validUntil` PR as that already contains breaking changes to the DateRegexp.